primitives/core/src/crypto.rs

```diff
@@ -355,282 +351,34 @@ fn ss58hash(data: &[u8]) -> blake2_rfc::blake2b::Blake2bResult {
-	AresAccount =>
-		(34, "ares", "Ares Protocol, standard account (*25519).")
-	ValiuAccount =>
-		(35, "vln", "Valiu Liquidity Network mainnet, standard account (*25519).")
-	CentrifugeAccount =>
-		(36, "centrifuge", "Centrifuge Chain mainnet, standard account (*25519).")
-	NodleAccount =>
-		(37, "nodle", "Nodle Chain mainnet, standard account (*25519).")
-	KiltAccount =>
-		(38, "kilt", "KILT Chain mainnet, standard account (*25519).")
-	PolimecAccount =>
-		(41, "poli", "Polimec Chain mainnet, standard account (*25519).")
-	SubstrateAccount =>
-		(42, "substrate", "Any Substrate network, standard account (*25519).")
-	BareSecp256k1 =>
-		(43, "secp256k1", "Bare ECDSA SECP256k1 key.")
-	ChainXAccount =>
-		(44, "chainx", "ChainX mainnet, standard account (*25519).")
-	UniartsAccount =>
-		(45, "uniarts", "UniArts Chain mainnet, standard account (*25519).")
-	Reserved46 =>
-		(46, "reserved46", "Reserved for future use (46).")
-	Reserved47 =>
-		(47, "reserved47", "Reserved for future use (47).")
-	NeatcoinAccount =>
-		(48, "neatcoin", "Neatcoin mainnet, standard account (*25519).")
-	PicassoAccount =>
-		(49, "picasso", "Composable Canary Network, standard account (*25519).")
-	ComposableAccount =>
-		(50, "composable", "Composable mainnet, standard account (*25519).")
-	HydraDXAccount =>
-		(63, "hydradx", "HydraDX standard account (*25519).")
-	AventusAccount =>
-		(65, "aventus", "Aventus Chain mainnet, standard account (*25519).")
-	CrustAccount =>
-		(66, "crust", "Crust Network, standard account (*25519).")
-	EquilibriumAccount =>
-		(67, "equilibrium", "Equilibrium Network, standard account (*25519).")
-	SoraAccount =>
-		(69, "sora", "SORA Network, standard account (*25519).")
-  ZeitgeistAccount =>
-		(73, "zeitgeist", "Zeitgeist network, standard account (*25519).")
-	MantaAccount =>
-		(77, "manta", "Manta Network, standard account (*25519).")
-	CalamariAccount =>
-		(78, "calamari", "Manta Canary Network, standard account (*25519).")
-	Polkadex =>
-		(88, "polkadex", "Polkadex Mainnet, standard account (*25519).")
-	PolkaSmith =>
-		(98, "polkasmith", "PolkaSmith Canary Network, standard account (*25519).")
-	PolkaFoundry =>
-		(99, "polkafoundry", "PolkaFoundry Network, standard account (*25519).")
-  OriginTrailAccount =>
-		(101, "origintrail-parachain", "OriginTrail Parachain, ethereumm account (ECDSA).")
-	HeikoAccount =>
-		(110, "heiko", "Heiko, session key (*25519).")
-	CloverAccount =>
-		(128, "clover", "Clover Finance, standard account (*25519).")
-	ParallelAccount =>
-		(172, "parallel", "Parallel, session key (*25519).")
-	SocialAccount =>
-		(252, "social-network", "Social Network, standard account (*25519).")
-	Moonbeam =>
-		(1284, "moonbeam", "Moonbeam, session key (*25519).")
-	Moonriver =>
-		(1285, "moonriver", "Moonriver, session key (*25519).")
-	BasiliskAccount =>
-		(10041, "basilisk", "Basilisk standard account (*25519).")
-
-	// Note: 16384 and above are reserved.
```

Comment:
here it says 16384 and above are reserved, but in the is_reserved implementation it is not considered reserved. (both before and after this PR)

Reply:
Good question. They were reserved here: https://github.com/paritytech/substrate/pull/5189/files - I think those specific ones are reserved for future `bare` keys.

The reserved above number was bumped from 48 to 16384 here: https://github.com/paritytech/substrate/pull/8165/files . I think it's reasonable to return true for is_reserved above 16384 for now. Plenty of headroom.

Reply:
I've made everything above 16384 be reserved.